sdk/storage/ci.yml

```diff
@@ -1,4 +1,4 @@
-# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
@danieljurek  is there a way to validate in CI that the yml file being modified is correct, so it fails CI (without having to trigger this pipeline manually).